corehq/apps/builds/templates/builds/edit_menu.html

```diff
@@ -1,23 +1,35 @@
 {% extends 'style/base_page.html' %}
 {% load hq_shared_tags %}
 {% load compress %}
+{% load i18n %}
+
 
 {% block js-inline %}{{ block.super }}
     <script>
         var doc = {{ doc|JSON }};
         var pretty_doc = JSON.stringify(doc, undefined, 2)
 
-        function Version(version, label, superuser_only) {
+        function Version(version, label, superuser_only, j2me_enabled) {
             var self = this;
             self.version = ko.observable(version);
             self.label = ko.observable(label);
             self.superuser_only = ko.observable(superuser_only);
+            self.j2me_enabled = ko.observable(j2me_enabled);
+
+            // subscribe for change in version to update j2me_enabled
+            // property and hence the checkbox in view
+            self.version.subscribe(function(newValue){
+                self.j2me_enabled(buildsMenu.j2me_enabled_versions.
+                                  includes(newValue)
+                                 );
```

Comment:
This subscribe here is so when you change the version in the Version dropdown, the check mark also changes, right? Nicely done.